include/pnet_api.h

```diff
@@ -33,11 +33,13 @@ extern "C"
 {
 #endif
 
-#include <stdint.h>
-#include <stddef.h>
+#include <pnet_export.h>
+#include "osal.h"
```

Comment:
See other comment - pnet_api.h should not depend on osal.h.

Either add an internal os_ethaddr_t type or just change the osal mac api:s to use standard types, e.g.:
```

/**
 * Read the MAC address.
 *
 * @param interface_name      In: Name of network interface
 * @param octet1              Out: The first (in transmission order) octet
 * @param octet2              Out: The second octet
 * @param octet3              Out: The third octet
 * @param octet4              Out: The fourth octet
 * @param octet5              Out: The fifth octet
 * @param octet6              Out: The sixth octet
 *
 * @return 0 on success and
 *         -1 if an error occurred
*/
int os_get_macaddress(
   const char              *interface_name,
   uint8_t                 *octet1,
   uint8_t                 *octet2,
   uint8_t                 *octet3,
   uint8_t                 *octet4,
   uint8_t                 *octet5,
   uint8_t                 *octet6
);

```

Reply:
Did put back pnet_ethaddr_t, and use a os_ethaddr_t